src/pages/index.js

```diff
@@ -53,29 +54,89 @@ const api = new Api({
   },
 });
 
+api
+  .getApiInfo()
+  .then(([userData, userCards]) => {
+    userId = userData._id;
+    console.log(userId);
+    userInfo.setUserInfo(userData);
+    userInfo.setAvatar(userData);
+    cardSection = new Section(
+      { items: userCards, renderer: createCard },
+      containerSelectors.cardSection
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+
+/* -------------------------------- user info ------------------------------- */
+
 const userInfo = new UserInfo(profileTitle, profileDescription, profileImage);
 
+/* -------------------------------- edit form ------------------------------- */
+
 const editFormPopup = new PopupWithForm(
   containerSelectors.profilePopup,
-  submitEditProfile
+  (inputValues) => {
+    editFormPopup.renderLoading(true);
+
+    return api
+      .editUserInfo(inputValues.title, inputValues.description)
+      .then(() => {
+        userInfo.setUserInfo({
+          name: inputValues.title,
+          about: inputValues.description,
+        });
+        editFormPopup.close();
+      })
+      .catch((err) => {
+        console.log(err);
+      })
+      .finally(() => {
+        editFormPopup.renderLoading(false, "Save");
+      });
+  }
 );
 
-const addFormPopup = new PopupWithForm(
+/* ----------------------------- add card popup ----------------------------- */
+
+const addCardPopup = new PopupWithForm(
   containerSelectors.addCardPopup,
-  (name, link) => {
-    addFormPopup.renderLoading(true);
+  (data) => {
+    addCardPopup.renderLoading(true);
     api
-      .addNewCard(name, link)
+      .addNewCard(data.title, data.link)
       .then((data) => {
-        const card = createCard(data);
+        createCard(data);
         addCardPopup.close();
-        cardSection.addItem(card.renderCard());
       })
       .catch((err) => {
         console.log(err);
       })
       .finally(() => {
-        addFormPopup.renderLoading(false, "Create");
+        addCardPopup.renderLoading(false, "Create");
+      });
+  }
+);
+
+const avatarPopup = new PopupWithForm(
+  containerSelectors.avatarPopup,
+  (data) => {
+    avatarPopup.renderLoading(true);
+    api
+      .updateProfileAvatar(data.link)
+      .then((res) => {
+        userInfo.setUserInfo(res);
```

Comment:
`        userInfo.setUserInfo(res);
`

You don't need this here. 

You need to set only `avatar`